test/src/test/java/hudson/util/BootFailureTest.java

```diff
@@ -48,6 +48,7 @@ public void before() throws Throwable {
 
         @Override
         public Hudson newHudson() throws Exception {
+            localPort = 0;
```

Comment:
https://github.com/jenkinsci/jenkins-test-harness/pull/132 causes `CustomRule` to reuse the port causing a `BindException`. This resets the port whenever `newHudson()` is invoked. See https://github.com/jenkinsci/jenkins-test-harness/pull/132/files#diff-e9c3a8021f501e45bfabdf6b4d0d2f5aR315